cart/cart.py

```diff
@@ -0,0 +1,53 @@
+from decimal import Decimal
```

Comment:
is this file for CRUD operation?

Reply:
yes, it has been made especially for cart updation and deletion